src/tasks/WasmBuildTasks/GetChromeVersions.cs

```diff
@@ -9,14 +9,15 @@
 using System.Net.Http;
 using System.Net;
 using System.Text.Json;
+using System.Text.RegularExpressions;
 using System.Threading.Tasks;
 using Microsoft.Build.Framework;
 
 using MBU = Microsoft.Build.Utilities;
 
 namespace Microsoft.WebAssembly.Build.Tasks;
 
-public class GetChromeVersions : MBU.Task
+public partial class GetChromeVersions : MBU.Task
```

Comment:
Where is the other part of this partial class?
Edit: okay, I see it's a requirement of `GeneratedRegexAttribute` to have it partial.